From: 18978327006 <1907810139@qq.com>
Date: Fri, 11 Apr 2025 03:39:31 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (153 of 153 strings)

Translation: openkylin-nile-new/kylin-connectivity
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-connectivity/vi/
---
 translations/kylin-connectivity_vi.ts | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/translations/kylin-connectivity_vi.ts b/translations/kylin-connectivity_vi.ts
index 335fe11..4b4fcc5 100644
--- a/translations/kylin-connectivity_vi.ts
+++ b/translations/kylin-connectivity_vi.ts
@@ -163,7 +163,7 @@
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="106"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="626"/>
         <source>List Mode</source>
-        <translation>Kiểu mẫu danh sách</translation>
+        <translation>Chế độ danh sách</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="118"/>
@@ -196,12 +196,12 @@
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="649"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="657"/>
         <source>Deselect All</source>
-        <translation>Bỏ chọn tất cả</translation>
+        <translation>Hủy chọn tất cả</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="631"/>
         <source>Icon Mode</source>
-        <translation>Kiểu mẫu biểu tượng</translation>
+        <translation>Chế độ biểu tượng</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="91"/>
@@ -250,7 +250,7 @@
         <location filename="../ui/filemanageview/fileview.cpp" line="132"/>
         <location filename="../ui/filemanageview/fileview.cpp" line="525"/>
         <source>Download</source>
-        <translation>Tải xuống thiết bị</translation>
+        <translation>Tải xuống đến cục bộ</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/fileview.cpp" line="405"/>
@@ -273,7 +273,7 @@
     <message>
         <location filename="../ui/view/homepage.cpp" line="48"/>
         <source>Cross equipment and cross system collaboration. It is more convenient to share resources and screens, and more convenient and efficient to work!</source>
-        <translation>Đồng bộ đa thiết bị,đa nền tảng,chia sẻ tệp,màn hình thời gian thực,tối ưu hóa hiệu suất công việc！</translation>
+        <translation>Phối hợp đồng bộ đa thiết bị,đa nền tảng,chia sẻ tệp,màn hình trong thời gian thực, tạo ra cách làm việc hiệu quả！</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="55"/>
@@ -327,7 +327,7 @@
     <message>
         <location filename="../ui/mainwindow.cpp" line="483"/>
         <source>Establishing connection, please wait...</source>
-        <translation>Đang thiết lập kết nối,vui lòng chờ</translation>
+        <translation>Đang thiết lập kết nối,vui lòng chờ đợi</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="487"/>
@@ -359,7 +359,7 @@
     <message>
         <location filename="../ui/mainwindow.cpp" line="1083"/>
         <source>After consent, the other party can view and download all the files on the device, and can share the other party&apos;s desktop to this screen.</source>
-        <translation>Sau khi đồng ý,đối phương có thể xem,tải xuống mọi tệp trên thiết bị này, và chia sẻ màn hình của họ lên màn hình hiện tại.</translation>
+        <translation>Sau khi đồng ý, bên kia có thể xem, tải xuống tất cả các tệp trên thiết bị này, và chia sẻ màn hình nền của bên kia trên màn hình này.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1089"/>
@@ -376,17 +376,17 @@
     <message>
         <location filename="../ui/mainwindow.cpp" line="1346"/>
         <source>File open exception!</source>
-        <translation>Lỗi mở tệp!</translation>
+        <translation>Tệp mở bất thường!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1423"/>
         <source>The other party has refused your screen projection request!</source>
-        <translation>Đối phương từ chối yêu cầu chia sẻ màn hình của bạn！</translation>
+        <translation>Bên kia đã từ chối yêu cầu chiếu màn hình của bạn!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1424"/>
         <source>Failed to cast the screen. Please contact the other party and try again.</source>
-        <translation>Chiếu màn hình thất bại,vui lòng liên hệ đối phương và thử lại.</translation>
+        <translation>Chiếu màn hình thất bại,vui lòng liên hệ với bên kia và thử lại.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1108"/>
@@ -397,12 +397,12 @@
     <message>
         <location filename="../ui/mainwindow.cpp" line="1103"/>
         <source>The other party has rejected your connection request!</source>
-        <translation>Đối phương đã từ chối yêu cầu kết nối của bạn！</translation>
+        <translation>Bên kia đã từ chối yêu cầu kết nối của bạn！</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1104"/>
         <source>Connection failed. Please contact the other party and try again.</source>
-        <translation>Kết nối thất bại,vui lòng liên hệ đối phương và thử lại.</translation>
+        <translation>Kết nối thất bại,vui lòng liên hệ với bên kia và thử lại.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1109"/>
@@ -416,12 +416,12 @@
     <message>
         <location filename="../ui/mainwindow.cpp" line="1207"/>
         <source>Please install kylin-assistant on the Android terminal!</source>
-        <translation>Vui lòng tải ứng dụng Trợ lý Kỳ Lân（Kylin-assistant） trên điện thoại di động！</translation>
+        <translation>Vui lòng tải xuống ứng dụng Trợ lý Kỳ Lân（Kylin-assistant） trên điện thoại di động！</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1217"/>
         <source>Please use the USB to connect your phone device!</source>
-        <translation>Vui lòng kết nối thiết bị điện thoại bằng cáp USB</translation>
+        <translation>Vui lòng kết nối thiết bị điện thoại bằng USB</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1228"/>
